domains/transfer/Transfer.API/API/Transfer/Api/v2024_01_03/Controllers/TransferAgreementProposalController.cs

```diff
@@ -19,26 +19,14 @@ namespace API.Transfer.Api.v2024_01_03.Controllers;
 [Authorize]
 [ApiController]
 [ApiVersion("20240103")]
-[Route("api/transfer-agreement-proposals")]
```

Comment:
Has Alexander already migrated to front-end to the new endpoints?

Reply:
Won't this break the frontend then?

Reply:
Most likely.